### PR TITLE
[EWL-4653] - Update Tools header

### PR DESCRIPTION
**Pull request by nstriedinger**
## Ticket(s)

**Jira Ticket**
- https://issues.ama-assn.org/browse/EWL-4653

## Description
Update Tools header to H4. PR BOTH SG2 and D8 elements.

## D8
- https://github.com/AmericanMedicalAssociation/ama-d8/pull/352

## To Test
- [ ] Visit http://localhost:3000/?p=organisms-tools and verify Tools header is H4
- [ ] Visit http://localhost:3000/?p=pages-topic and verify Tools header is H4
- [ ] Visit http://localhost:3000/?p=pages-news and verify Tools header is H4
- [ ] Visit http://localhost:3000/?p=pages-press-release and verify Tools header is H4

## Visual Regressions
N/A

## Relevant Screenshots/GIFs
Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.
screen shot 2018-04-03 at 1 13 05 pm

## Remaining Tasks
N/A

## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

File: styleguide/source/_patterns/03-organisms/tools.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "tools": {
     "heading": {
-      "class": "ama__h5",
+      "class": "ama__h4",
```
Comment (labbydev) [inline styleguide/source/_patterns/03-organisms/tools.json:4]: Should we be adding the classes in the twig instead of the json?

**Review by labbydev — COMMENTED** [review]
This all looks good to me, I'm just curious if the classes should be in the json or the twig. I'd like to get @designerneil and @chelsiejohnston thoughts on this.

File: styleguide/source/_patterns/03-organisms/tools.json (hunk 1 above)
Comment (nstriedinger) [reply styleguide/source/_patterns/03-organisms/tools.json:4]: I'm not sure what we decided on.

Comment (nstriedinger) [reply styleguide/source/_patterns/03-organisms/tools.json:4]: @labbydev Neil thinks that it doesn't matter.

Comment (chelsiejohnston) [reply styleguide/source/_patterns/03-organisms/tools.json:4]: @nstriedinger @labbydev My opinion is that we should be adding them in the Twig whenever possible.

Comment (labbydev) [reply styleguide/source/_patterns/03-organisms/tools.json:4]: According to FED meeting notes from 3/20/18
> Headings will continue to leave the class in the Twig

**Pull request merged**